docs/cookbook/keys-descriptors/descriptors.md

```diff
@@ -1,4 +1,4 @@
-# Working with Descriptors
+# Creating Seeds, Keys, and Descriptors
```

Comment:
I think this page should keep it's original title and intent (although I also like your title in the mkdocs file `Creating Descriptors`.

I suggest keeping the code we currently have for this page too (although printing the public descriptor is a good addition).

Reply:
ok, ya one thing here is to clarify the distinction btw pubkey and privkey descriptors. using seed phrase means we don't have to address that. but we can use the quickstart code with privkey descriptors for the full-wallet purposes... just need to note that I'd say